typescript/infra/helm/check-warp-deploy/templates/_helpers.tpl

```diff
@@ -0,0 +1,42 @@
+{{/*
```

Comment:
@tkporter - lifted most of this form key funder

Reply:
Not suggesting to do this now but some of these helm files would be nice to live in a common library that all the charts consume from. This and the external secrets stuff to avoid duplication